src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -230,6 +230,33 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
         }
 
+        /// <summary>
+        /// Gets an &lt;input&gt; elements "type" attribute value for the given <paramref name="modelExplorer"/>.
```

Comment:
No, `Gets an &lt;input&gt; element's "type" attribute value based the given <paramref name="modelExplorer"/> or <see cref="InputType"/>.` Note the apostrophe as well as the property mention.